controllers/rhmi/bootstrapReconciler.go

```diff
@@ -319,25 +281,23 @@ func (r *Reconciler) setTenantMetrics(ctx context.Context, serverClient k8sclien
 }
 
 func (r *Reconciler) reconcilePriorityClass(ctx context.Context, serverClient k8sclient.Client) (integreatlyv1alpha1.StatusPhase, error) {
-	if integreatlyv1alpha1.IsRHOAM(rhmiv1alpha1.InstallationType(r.installation.Spec.Type)) {
-		priorityClass := &schedulingv1.PriorityClass{
-			ObjectMeta: metav1.ObjectMeta{
-				Name: r.installation.Spec.PriorityClassName,
-			},
+	priorityClass := &schedulingv1.PriorityClass{
+		ObjectMeta: metav1.ObjectMeta{
+			Name: r.installation.Spec.PriorityClassName,
+		},
+	}
+	if _, err := controllerutil.CreateOrUpdate(ctx, serverClient, priorityClass, func() error {
+		if integreatlyv1alpha1.IsRHOAMMultitenant(rhmiv1alpha1.InstallationType(r.installation.Spec.Type)) {
+			priorityClass.Value = 0
+		} else {
+			priorityClass.Value = 1000000000
 		}
-		if _, err := controllerutil.CreateOrUpdate(ctx, serverClient, priorityClass, func() error {
-			if integreatlyv1alpha1.IsRHOAMMultitenant(rhmiv1alpha1.InstallationType(r.installation.Spec.Type)) {
-				priorityClass.Value = 0
-			} else {
-				priorityClass.Value = 1000000000
-			}
-			priorityClass.GlobalDefault = false
-			priorityClass.Description = "Priority Class for managed-api"
+		priorityClass.GlobalDefault = false
+		priorityClass.Description = "Priority Class for managed-api"
 
-			return nil
-		}); err != nil {
-			return integreatlyv1alpha1.PhaseInProgress, err
-		}
+		return nil
+	}); err != nil {
+		return integreatlyv1alpha1.PhaseInProgress, err
```

Comment:
Would phaseFailed, err make more sense here?

Reply:
As this was the existing behaviour I am reluctant to change it now. There are other functions that give the same `PhaseInProgress` return value on error. If one is to be changed they should all be changed. There are also no unit tests checking this function. The removal of RHMI is not driving the need for this change.

Reply:
No strong opinion, but maybe a follow up Jira to double check if our returns of phases make sense would be good here